test/test_lava_test_plans.py

```diff
@@ -121,3 +121,25 @@ def test_call_lava_test_plan_testplans_ti(param):
     variable_input_file, device, testplan = param
     sys.argv = shlex.split(f'lava_test_plans --dry-run --variables "{variable_input_file}" --device-type "{device}" --test-plan "{testplan}" {test_lava_validity}')
     assert main() == 0, f"fail: {sys.argv}"
+
+# lkft-android tests
+
+lkft_android_project_device_path = "lava_test_plans/projects/lkft-android/devices"
+lkft_android_project_testpaln_path = "lava_test_plans/projects/lkft-android/"
```

Comment:
This isn't needed. please drop.

Reply:
testpaln is necessary, there will be lkft-android specific testcases added there later.